targetselector/running_target_selector.go

```diff
@@ -87,15 +87,15 @@ func (r runningTargetSelector) GetMatchingPods(c client.Client, instance *chaosv
 		}
 	}
 
-	return runningPods, nil
+	return runningPods, len(pods.Items), nil
 }
 
 // GetMatchingNodes returns a nodes list containing all nodes matching the given label selector
```

Comment:
I know you updated the godoc comment in `target_selector.go`, but could you also edit the two comments here to explain what the `int` return is for?